python/feldera/rest/feldera_client.py

```diff
@@ -491,6 +558,14 @@ def pause_pipeline(
 
         self.__wait_for_pipeline_state(pipeline_name, "paused", timeout_s)
 
+    def approve_pipeline(

```

Comment:
maybe `approve_bootstrap` is better; this name could mean something else later

Reply:
I will probably stick with `approve`. We may need to tune the wording and documentation, but you're generally approving pipeline changes, which may or my not involve bootstrapping. In general you're approving the pipeline to run, for which `/approve` seems adequate.